config/api.yml

```
@@ -847,6 +847,7 @@
     :verbs: *gpppd
     :subcollections:
     - :tags
+    - :snapshots
```

```suggestion
    - :cloud_volume_snapshots
```

This is going to link to the wrong class